packages/reactive-element/src/css-tag.ts

```diff
@@ -15,21 +15,36 @@ export const supportsAdoptingStyleSheets =
   'adoptedStyleSheets' in Document.prototype &&
   'replace' in CSSStyleSheet.prototype;
 
+/**
+ * A CSSResult or native CSSStyleSheet.
+ *
+ * In browsers that support constructible CSS style sheets, CSSStyleSheet
+ * object can be used for styling along side CSSResult from the `css`
+ * template tag.
+ */
 export type CSSResultOrNative = CSSResult | CSSStyleSheet;
 
-export type CSSResultFlatArray = CSSResultOrNative[];
-
-export type CSSResultArray = Array<CSSResultOrNative | CSSResultArray>;
```

Comment:
`CSSResultArray` is exported in the old version and there are people using this internally - not tons though.

https://github.com/lit/lit-element/blob/2b398727dacb7694fbf38816824675d20838704f/src/lit-element.ts#L82-L83

Reply:
oh... I thought it was new. I'll add it back.